code/modules/mob/living/carbon/human/human_movement.dm

```diff
@@ -12,6 +12,8 @@
 	if(embedded_flag)
 		handle_embedded_objects() //Moving with objects stuck in you can cause bad times.
 
+	if(slowed)

```

Comment:
if you're making this at a living level you need to implement this in living/movement_delay() as well